test/functional/ui/popupmenu_spec.lua

```diff
@@ -2488,6 +2538,11 @@ describe('builtin popupmenu', function()
         end
         feed('<C-E><ESC>')
       end)
+
+      it('bufnr of popup floating window does not changed', function()
```

Comment:
Is this about the "info" window? "popup floating window" could refer to either the menu itself or the flyout "info" window. Which is it?

is this a correct description?:
```suggestion
      it('popup info window reuses bufnr', function()
```

Reply:
yes clear description